frontend/src/contexts/authStateContext.tsx

```diff
@@ -1,5 +1,4 @@
 import * as React from 'react';
-import { useKeycloak } from '@react-keycloak/web';
```

Comment:
I was fairly certain when I removed this something broke.  I didn't understand why as it doesn't appear to be used in the file.